plugins/sdexFeed.go

```diff
@@ -0,0 +1,79 @@
+package plugins
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/interstellar/kelp/api"
+	"github.com/interstellar/kelp/support/utils"
+	"github.com/stellar/go/build"
+	"github.com/stellar/go/clients/horizon"
+)
+
+// sdexFeed represents a pricefeed from the SDEX
+type sdexFeed struct {
+	sdex       *SDEX
+	assetBase  *horizon.Asset
+	assetQuote *horizon.Asset
+}
+
+// ensure that it implements PriceFeed
+var _ api.PriceFeed = &sdexFeed{}
+
+// newSDEXFeed creates a price feed from buysell's url fields
+func newSDEXFeed(sdex *SDEX, url string) (*sdexFeed, error) {
+	s := new(sdexFeed)
+	s.sdex = sdex
+	urlParts := strings.Split(url, "/")
+
+	baseURL := strings.Split(urlParts[0], ":")
+	baseCode := baseURL[0]
+	baseIssuer := baseURL[1]
+	baseConvert, e := parseAsset(baseCode, baseIssuer)
+	if e != nil {
+		return nil, fmt.Errorf("unable to convert base asset url to sdex asset")
+	}
+	s.assetBase = baseConvert
+
+	quoteURL := strings.Split(urlParts[1], ":")
+	quoteCode := quoteURL[0]
+	quoteIssuer := quoteURL[1]
+	quoteConvert, e := parseAsset(quoteCode, quoteIssuer)
+	if e != nil {
+		return nil, fmt.Errorf("unable to convert quote asset url to sdex asset")
+	}
+	s.assetQuote = quoteConvert
+	return s, nil
+}
+
+// GetPrice returns the SDEX mid price for the trading pair
+func (s *sdexFeed) GetPrice() (float64, error) {
+	orderBook, e := GetOrderBook(s.sdex.API, s.assetBase, s.assetQuote)
+	if e != nil {
+		return 0, fmt.Errorf("unable to get sdex price: %s", e)
+	}
+	bids := orderBook.Bids
+	topBidPrice := utils.PriceAsFloat(bids[0].Price)
```

Comment:
can you add empty lines here: after error check, after topBidPrice, after lowAskPrice?